runtime/domain.c

```diff
@@ -450,10 +450,6 @@ static void create_domain(uintnat initial_minor_heap_wsize) {
     domain_state->trap_sp_off = 1;
 #endif
 
-#if defined(NAKED_POINTERS_CHECKER) && !defined(_WIN32)
-    domain_state->checking_pointer_pc = NULL;
```

Comment:
I don't see it being removed from `domain_state` as well, I think it can be removed, can it?